From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Helvetica Volubi <suisuroru@blue-millennium.fun>
Date: Mon, 10 Aug 2026 03:32:54 +0800
Subject: [PATCH] Rebrand to Lecithin


diff --git a/src/main/java/io/papermc/paper/ServerBuildInfo.java b/src/main/java/io/papermc/paper/ServerBuildInfo.java
index d9127bc6c5801521a5dd1171e4e215f241934034..9f821b5538f191d6b9af36e07f531ee0090784d1 100644
--- a/src/main/java/io/papermc/paper/ServerBuildInfo.java
+++ b/src/main/java/io/papermc/paper/ServerBuildInfo.java
@@ -28,6 +28,13 @@ public interface ServerBuildInfo {
     Key BRAND_LOPHINE_ID = Key.key("lophinelabs", "lophine");
     // Lophine end
 
+    // Lecithin start
+    /**
+     * The brand id for Lecithin.
+     */
+    Key BRAND_LECITHIN_ID = Key.key("lophinelabs", "lecithin");
+    // Lecithin end
+
     /**
      * Gets the {@code ServerBuildInfo}.
      *
