From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Helvetica Volubi <suisuroru@blue-millennium.fun>
Date: Mon, 10 Aug 2026 03:25:26 +0800
Subject: [PATCH] Force disable all unsupported check


diff --git a/src/main/java/me/earthme/luminol/config/modules/unsupported/DisableCheckForFoliaSupported.java b/src/main/java/me/earthme/luminol/config/modules/unsupported/DisableCheckForFoliaSupported.java
index 49638c4887492fedbab8b6ebe859790989e28fec..13712f5bc49ba38f813b424e2a5fadfa736456dd 100644
--- a/src/main/java/me/earthme/luminol/config/modules/unsupported/DisableCheckForFoliaSupported.java
+++ b/src/main/java/me/earthme/luminol/config/modules/unsupported/DisableCheckForFoliaSupported.java
@@ -4,11 +4,7 @@ import me.earthme.luminol.config.flags.ConfigClassInfo;
 import me.earthme.luminol.config.flags.ConfigInfo;
 import me.earthme.luminol.enums.EnumConfigCategory;
 
-@ConfigClassInfo(category = EnumConfigCategory.UNSUPPORTED, name = "disable_check_for_folia_supported")
 public class DisableCheckForFoliaSupported {
-    @ConfigInfo(name = "disable_for_paper")
-    public static boolean disableForPaper = false;
-
-    @ConfigInfo(name = "disable_for_leaves")
-    public static boolean disableForLeaves = false;
+    public static boolean disableForPaper = true;
+    public static boolean disableForLeaves = true;
 }
