From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Helvetica Volubi <suisuroru@blue-millennium.fun>
Date: Mon, 10 Aug 2026 03:10:35 +0800
Subject: [PATCH] Register config to config system


diff --git a/src/main/java/me/earthme/luminol/config/ConfigManager.java b/src/main/java/me/earthme/luminol/config/ConfigManager.java
index 7a9907727fb221e06ae1bb0a553ef96e94b6702c..892e255be6ef0a9a11918d39d31484dc45b7bbab 100644
--- a/src/main/java/me/earthme/luminol/config/ConfigManager.java
+++ b/src/main/java/me/earthme/luminol/config/ConfigManager.java
@@ -35,6 +35,7 @@ public class ConfigManager {
                 "fun.bm.lophine.carpet.config.modules"
         ));
         // Lophine end - add carpet config
+        registerConfig("lecithin", builder.of("lecithin", "fun.bm.lecithin.config.modules")); // Lecithin global config
         preLoad();
     }
 
