From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Helvetica Volubi <suisuroru@blue-millennium.fun>
Date: Fri, 21 Aug 2026 03:01:24 +0800
Subject: [PATCH] Register I18n


diff --git a/src/main/java/fun/bm/lophine/utils/ServerI18nUtil.java b/src/main/java/fun/bm/lophine/utils/ServerI18nUtil.java
index 907eef0ab2635d3f49f67649147ca29f003f9818..9a5af877cfc8559184772d3a0d612ba4f6ee80b5 100644
--- a/src/main/java/fun/bm/lophine/utils/ServerI18nUtil.java
+++ b/src/main/java/fun/bm/lophine/utils/ServerI18nUtil.java
@@ -57,6 +57,7 @@ public class ServerI18nUtil {
     private static Set<String> register() {
         Set<String> paths = new HashSet<>();
         register(paths, "assets/lophine/lang");
+        register(paths, "assets/lecithin/lang"); // Lecithin - add self lang file
         return paths;
     }
 
