From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Helvetica Volubi <suisuroru@blue-millennium.fun>
Date: Fri, 21 Aug 2026 03:01:24 +0800
Subject: [PATCH] Register I18n


diff --git a/src/main/java/fun/bm/lophine/utils/ServerI18nUtil.java b/src/main/java/fun/bm/lophine/utils/ServerI18nUtil.java
index bfd4d51180b2603068174ad840062ff02500ad6f..75df6a807e91141e392e43c91eb6a5fec1b77628 100644
--- a/src/main/java/fun/bm/lophine/utils/ServerI18nUtil.java
+++ b/src/main/java/fun/bm/lophine/utils/ServerI18nUtil.java
@@ -59,6 +59,7 @@ public class ServerI18nUtil {
     private static Set<String> register() {
         Set<String> paths = new HashSet<>();
         register(paths, "assets/lophine/lang");
+        register(paths, "assets/lecithin/lang"); // Lecithin - add self lang file
         return paths;
     }
 
