From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Helvetica Volubi <suisuroru@blue-millennium.fun>
Date: Fri, 21 Aug 2026 03:01:24 +0800
Subject: [PATCH] Register I18n


diff --git a/src/main/java/fun/bm/lophine/utils/ServerI18nUtil.java b/src/main/java/fun/bm/lophine/utils/ServerI18nUtil.java
index bd3b7b6ab86af8e0f97bed9db2026d607f1c3a3d..a93b3a9453edbe3b67335f21d7ca62fe294bded6 100644
--- a/src/main/java/fun/bm/lophine/utils/ServerI18nUtil.java
+++ b/src/main/java/fun/bm/lophine/utils/ServerI18nUtil.java
@@ -56,6 +56,7 @@ public class ServerI18nUtil {
 
     private static void register() {
         register("assets/lophine/lang");
+        register("assets/lecithin/lang"); // Lecithin - add self lang file
     }
 
     private static void register(String base) {
